community/consistency-check/src/main/java/org/neo4j/consistency/checking/full/CountsBuilderDecorator.java

```diff
@@ -162,7 +163,7 @@ public void checkCounts( CountsAccessor counts, final ConsistencyReporter report
             public void visitNodeCount( int labelId, long count )
             {
                 nodeEntries.incrementAndGet();
-                reporter.forCounts( new CountsEntry( nodeKey( labelId ), count ), CHECK_NODE_COUNT );
+                reporter.forCounts( new CountsEntry().initialize( nodeKey( labelId ), count ), CHECK_NODE_COUNT );
```

Comment:
Syre. CountsEntry and the other synthetic record classes also got `initialize` methods for consistency with the other classes, but are not really needed. Those can most likely be removed and we can go back to using their constructors instead.